frontend/src/components/Modal/modal.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { MdOutlineEmail } from 'react-icons/md';
+import { FaRegUser } from 'react-icons/fa';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { SubscriberForm } from './LatestUpdate.types';
+import { fetchData } from '@utils';
+import { API_ENDPOINTS, SUBSCRIBER_FORM_SCHEMA } from '@constants';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { enqueueSnackbar } from 'notistack';
+
+interface ModalProps {
+  onClose: () => void;
+}
+
+const Modal: React.FC<ModalProps> = ({ onClose }) => {
```

Comment:
I guess these changes are not part of this PR. We need to change target branch

Reply:
Not sure how this came here ?

Reply:
I guess this is because this branch was created from `hero_section_cta` branch but here its targeting main. try changing the target branch of this PR

Reply:
Done, have a look please.